src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    display: flex;
+    padding-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    text-decoration: none;
+    height: 60px;
+    align-items: center;
+    text-transform: uppercase;
+    font-family: "Roboto", sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    color: $color-main-accent;
```

Comment:
Consider to add a smooth transition for hover effect. Add `transition: color 0.3s;`